ccd-definition/CaseEventToFields/DefendantResponseAdmitPartLRspec.json

```diff
@@ -186,7 +186,7 @@
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
     "PageID": "HowToAddEmploymentDetails",
-    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" AND defenceAdmitPartEmploymentTypeRequired=\"Yes\" AND respondToClaimAdmitPartEmploymentTypeLRspec CONTAINS \"EMPLOYED\"",
+    "PageShowCondition": "respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\" AND defenceAdmitPartPaymentTimeRouteRequired=\"IMMEDIATELY\" OR respondToClaimAdmitPartEmploymentTypeLRspec CONTAINS \"EMPLOYED\"",
```

Comment:
If I'm not mistaken, usual AND/OR priority is AND first, OR second, so then HowToAddEmploymentDetails will appear if employed, whatever the defenceAdmitPartPaymentTimeRouteRequired or respondent1ClaimResponseTypeForSpec. Have you checked?

Reply:
In this case `respondent1ClaimResponseTypeForSpec=\"PART_ADMISSION\"` needs to be before `AND` to be evaluated first. then only if it is true the remainder of the condition is being evaluated.

Reply:
After trying a toy POC of the A and B or C operation, I'm not sure this is reliable. I have asked in exui support but, for the time being, I think we should check this works as intended as completely as we can

Reply:
This has been reverted because the change was unnecessary caused due to an incorrect understanding of the screen flow in miro.